roles/sap_ha_install_anydb_ibmdb2/tasks/db2_hadr_pcmk_cluster_create.yml

```diff
@@ -126,41 +169,47 @@
         /db2/$ibmdb2_instance/sqllib/bin/db2cm -create -cluster -domain db2pcmkcluster -host $anydb_primary -publicEthernet $anydb_primary_nic -host $anydb_secondary -publicEthernet $anydb_secondary_nic -remote_cmd "ssh -q -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
       failed_when: not __sap_ha_install_anydb_ibmdb2_config1.rc == 0 and not 'make sure there is no cluster on any of the hosts' in __sap_ha_install_anydb_ibmdb2_config1.stdout
 
-    # db2cm script must run as root
-    # Setting to be performed on both Primary and Secondary node
-    - name: SAP HA AnyDB - IBM Db2 HADR - Primary Node - Update corosync totem token timeout after creating cluster
-      register: __sap_ha_install_anydb_ibmdb2_config1
-      # when:
-      #   - sap_ha_install_anydb_ibmdb2_hostname_primary == inventory_hostname_short
+    # Corosync setting to be performed on both Primary and Secondary node
+    # Reasons for noqa:
+    # - command-instead-of-module: Shell commands to match IBM Db2 documentation
+    # - no-changed-when: sed will not change anything if already changed
+    - name: SAP HA AnyDB - IBM Db2 HADR - Primary Node - Update corosync totem token timeout after creating cluster # noqa command-instead-of-module no-changed-when
+      register: __sap_ha_install_anydb_ibmdb2_config2
       ansible.builtin.shell: |
         sed -i 's/token: 10000/token: 30000/g' /etc/corosync/corosync.conf
```

Comment:
Consider using the lineinfile module for performing this change. Matching the IBM db2 documentation appears to be an insufficient justification for using a command instead of an Ansible module.